### ISSUE TITLE
Rename model validation set names

### ISSUE THREAD

**Issue by jboelaert** [emilienschultz/pyactivetigger#774]
In model validation, could we rename 

- in Models tab, "Macro F1 score on train-test set" -> "Macro F1 score on Train-Eval set"
- in Evaluation tab
  + "train_scores" -> "Train-All"
  + "outofsample_scores" -> "Train-Eval"
  + "valid_scores" -> "Validation set"
  + "test_scores" -> "Test set"
